### PR TITLE
Azure VN peering documentation

### PR DESCRIPTION
**Pull request by lennessyy**

### CODE REVIEW

File: modules/vpc-peering/pages/azure.adoc [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+From the platform list, click btn:[Initiate a VPC Peering].
+
+image::cloud-network.png[Network Tab]
+
+In the pop-up window, choose Azure as your network platform.
+Then choose the region of the platform you want to initiate the peering connection from.
+Click btn:[Next].
+
+=== 2. Add TigerGraph Cloud to your Azure tenant
+In the Peered VPC tab, fill in your link:https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-how-to-find-tenan[Azure tenant ID] and your VN ID (Resource ID for your VN).
```
Comment (vpoliakov01) [inline modules/vpc-peering/pages/azure.adoc:26]:
```suggestion
In the Peered VPC tab, fill in your link:https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-how-to-find-tenant[Azure tenant ID] and your VN ID (Resource ID for your VN).
```

**Review by vpoliakov01 — CHANGES_REQUESTED** [review]

File: modules/vpc-peering/pages/azure.adoc [hunk 2]
```diff
@@ -0,0 +1,48 @@
+= Set up VN Peering on Azure
+:description: The process to set up VPC peering between the TigerGraph Cloud Azure VN and another Azure VN.
+:experimental:
+
+
+This is a guide for setting up a Virtual Network(VN) peering connection between the TigerGraph Cloud VN and another Azure VN.
```
Comment (a-m-thomas) [inline modules/vpc-peering/pages/azure.adoc:6]: Space before the first parenthesis

File: modules/vpc-peering/pages/azure.adoc [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+From the list of roles, choose Network Contributor.
+Click btn:[Next].
+
+Click btn:[Select Members] and search for TigerGraph VN Peerer.
+Select and click btn:[Review + assign].
+You'll see a notification if the role assignment is successful.
+
+=== 4. Initiate peering
+Go back to the Cloud Portal, and click btn:[Initiate Peering].
+If connection is successful, you'll see the connection listed under Azure and its status is "Connected".
```
Comment (a-m-thomas) [inline modules/vpc-peering/pages/azure.adoc:48]: If the connection is successful, you'll see the connection listed under Azure with its status shown as "Connected".

**Review by alexthomasTG — COMMENTED** [review]
Clear and easy to follow!

File: modules/vpc-peering/pages/azure.adoc (hunk 2 above)
Comment (lennessyy) [reply modules/vpc-peering/pages/azure.adoc:6]: Thanks for spotting! Fixed

**Pull request merged**